packages/salesforcedx-vscode-core/test/vscode-integration/context/workspaceContext.test.ts

```diff
@@ -96,7 +99,7 @@ describe('WorkspaceContext', () => {
   const testAlias = 'TestOrg';
   const testUser2 = 'test2@test.com';
   const cliConfigPath = join(
-    getRootWorkspacePath(),
+    workspaceUtils.getRootWorkspacePath(),
```

Comment:
verified import statement and change in function calls due to refactoring in workspaceUtils.